src/styles/watch.scss

```diff
@@ -0,0 +1,44 @@
+.stopwatch {
+  position: relative;
+  width: $watch-size;
+  height: $watch-size;
+  border: 5px dotted $border-color;
+  border-radius: 50%;
+  background-color: #fff;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)